src/grids.cpp

```diff
@@ -1584,28 +1595,27 @@ NTv1Grid *NTv1Grid::open(PJ_CONTEXT *ctx, std::unique_ptr<File> fp,
     }
 
     ExtentAndRes extent;
-    extent.westLon = -to_double(header + 72) * DEG_TO_RAD;
-    extent.southLat = to_double(header + 24) * DEG_TO_RAD;
-    extent.eastLon = -to_double(header + 56) * DEG_TO_RAD;
-    extent.northLat = to_double(header + 40) * DEG_TO_RAD;
-    extent.resLon = to_double(header + 104) * DEG_TO_RAD;
-    extent.resLat = to_double(header + 88) * DEG_TO_RAD;
-    if (!(fabs(extent.westLon) <= 4 * M_PI &&
-          fabs(extent.eastLon) <= 4 * M_PI &&
-          fabs(extent.northLat) <= M_PI + 1e-5 &&
-          fabs(extent.southLat) <= M_PI + 1e-5 &&
-          extent.westLon < extent.eastLon &&
-          extent.southLat < extent.northLat && extent.resLon > 1e-10 &&
-          extent.resLat > 1e-10)) {
+    extent.isGeographic = true;
+    extent.west = -to_double(header + 72) * DEG_TO_RAD;
```

Comment:
Could these magic numbers ever change?

Reply:
NTv1, an obsolete format, changing ? No. Those are fixed offsets in the file